group_vars/all

```diff
@@ -3,9 +3,14 @@
 nfs_server_control_room: 10.128.255.9
 nfs_server_control_opi: 10.128.255.9
 
-fac_workstation_ip: 10.0.7.55
+nfs_clients_auto_backup_ta_ti_rack_ip: 10.128.255.5
+nfs_clients_auto_backup_ca_ra_ctrl_ip: 10.128.255.4
+nfs_clients_auto_backup_la_ra_ctrl_ip: "10.128.255.3
```

Comment:
Is `"` a typo?

Reply:
Yes.... Thanks for catching that. I didn't test this role as I didn't want to change anything at the server, but I will test it now on m machine. Thanks again!